pkg/actions/actions.go

```diff
@@ -325,7 +324,7 @@ func (o *optionLoader) LoadClientConfig() *client.Config {
 func (o *optionLoader) LoadApp() app.App {
 	i := o.load(OptionApp)
 	if i == nil {
-		o.err = ErrNotInApp
+		o.err = ErrInvalidApp
```

Comment:
This means we lose visibility into the root cause -  is this preferable to `errors.Wrap(o.err, ...)`?